### DISCUSSION TITLE
Problem: When packaging NiceGui App with PyInstaller. RunTime Error Static Does not exist.

### DISCUSSION THREAD

**Discussion by rehanshehzi** [zauberzeug/nicegui#1135] [Q&A]
### Question

I followed the steps from 
https://nicegui.io/documentation#package_for_installation

`ui.run(reload=False)
`
Also added the following in 

```
a = Analysis(
    ['..\\awesome_app\\main.py'],
    pathex=[],
    binaries=[],
    datas=[("..\\..\\.venv\\lib\\site-packages\\nicegui", 'nicegui')],
```

The executable gets package. But when i double click it,  i get the following error 
![niceGui Runtime Error](https://github.com/zauberzeug/nicegui/assets/1922215/ae249c77-4f1a-4ec3-9c9c-432536c07af9)

**One insteresting observation is that the nicegui folder doesnt get copied; so that is empty **

Thanks in advance ~~

Best regards,
Rehan

**Comment by falkoschindler** [comment]
Hi @rehanshehzi,
Unfortunately I have no idea what is happening there. What is `Analysis` actually?
It would help a lot if you could show a [minimal reproducible code example](https://en.wikipedia.org/wiki/Minimal_reproducible_example) that is causing the error.

**Comment by rehanshehzi** [comment]
Hi, 
Thanks for your reply. 
I am editting the main question

**Comment by CatamountJack** [comment]
Two thoughts...  since you said the nicegui folder is empty, try freezing using an absolute path to your `\\site-packages\\nicegui` folder, just to be sure it's not a relative path issue.  Also, are you adding static local files in your app?  If you're using something like `app.add_static_files('/static', 'static')`, you'll want to also include that folder in the `datas` directive of the `Analysis` section in your PyInstaller spec file.  In your case, perhaps something like:

```
datas=[("\\absolute_path\\lib\\site-packages\\nicegui", 'nicegui'),
       (".\\static", "static"),
       ],
```
